### DISCUSSION TITLE
Multiple Sort Levels in Preview Data

### DISCUSSION THREAD

**Discussion by cygnex** [TabularEditor/TabularEditor3#219] [🙋♀️TE3 Q&A] [answered]
Is there a way to have multiple levels of sorting in Preview Data?

For example, I want to sort by State and then by Cities in those States.

AK   |  Anchorage
AK   |  Anchorage
AK   |  Fairbanks
AK   |  Juneau
AK   |  Juneau

**Comment by otykier** [comment, marked as answer]
Until recently, this was not possible, since Tabular Editor uses the [TOPNSKIP](https://dax.guide/topnskip) DAX function to preview a small number of rows from the table. In earlier versions of Analysis Services TOPNSKIP only supported a single Order By argument, but this has recently been changed, so we will consider adding support for multiple-sort levels in a future update of TE3.
